### PR TITLE
Delete Mendix 7 and Studio Pro 7 instances

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/en/docs/appstore/use-content/partner-solutions/apd/rg-apd/rg-two-apm/rg-two-recorder/rg-two-browser-recorder-results.md [hunk 1]
```diff
@@ -55,10 +55,6 @@ The response can include the number of objects retrieved, the count of an XPath,
 
 The **Connection** tab displays the header information and the timings of the request. The timing of a request includes the time needed to send the response to the server, the wait time due to server processing, and the time needed to download the response from the server. 
 
-{{% alert color="info" %}}
```
Comment (MariaShaposhnikova) [inline content/en/docs/appstore/use-content/partner-solutions/apd/rg-apd/rg-two-apm/rg-two-recorder/rg-two-browser-recorder-results.md:58]: @Luyao-Zhang-1 , Is it fine to delete this? Is it planned to make it available? I checked the screenshot and did not see Request timing information there, have I missed it?

File: content/en/docs/appstore/use-content/partner-solutions/apd/rg-apd/rg-three-apd/rg-three-recorder/rg-three-browser-recorder-results.md [hunk 2]
```diff
@@ -55,10 +55,6 @@ The response can include the number of objects retrieved, the count of an XPath,
 
 The **Connection** tab displays the header information and the timings of the request. The timing of a request includes the time needed to send the response to the server, the wait time due to server processing, and the time needed to download the response from the server. 
 
-{{% alert color="warning" %}}
```
Comment (MariaShaposhnikova) [inline content/en/docs/appstore/use-content/partner-solutions/apd/rg-apd/rg-three-apd/rg-three-recorder/rg-three-browser-recorder-results.md:58]: @Luyao-Zhang-1, Is it fine to delete this? Is it planned to make it available? I checked the screenshot and did not see Request timing information there, have I missed it?

File: content/en/docs/control-center/security/set-up-sso-byoidp.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ BYOIDP SSO has the following limitations.
 * If the user's email address is changed in your IdP, Mendix may not recognize it as the same account and will ask the user to set up a new Mendix account.
 * `login_hint` is not optional and is always sent as part of authentication requests to the IdP.
 * BYOIDP SSO only supports OIDC and does not support other protocols such as SAML.
-* Your Mendix app must be built using Mendix 7.23 or above.
+* Your Mendix app must be built using Mendix 7.23 or above. <!--Still valid?-->
```
Comment (MariaShaposhnikova) [inline content/en/docs/control-center/security/set-up-sso-byoidp.md]: @MarkvanMents , shall we keep this info about 7?

File: content/en/docs/deployment/mobileapp.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ In the **General settings** section, you must provide the following information:
 
 ### 2.2 Profile Settings
 
-If you are targeting Mendix 7.2.0 or above, please specify the navigation profiles to use on phones and tablets.
+If you are targeting Mendix 7.2.0 or above, please specify the navigation profiles to use on phones and tablets. <!--Should we rewrite this?-->
```
Comment (MariaShaposhnikova) [inline content/en/docs/deployment/mobileapp.md]: @dbreseman or @ConnorLand , can I delete "If you are targeting Mendix 7.2.0 or above"?

File: content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ Follow these steps to export the Mendix components and store them on your local
 1. Start the Configuration Tool using the command `mxpc-cli registry-migration` to initiate registry migration mode.
 2. Select **Migration Type** to be *Export*.
 3. Select the **Main Components** and **Storage Provisioners** you want to export.
-4. Enter the version numbers of any Mendix Runtimes which you want to export. You can use wildcards and create a list of versions separated by commas. For example `8.12.*, 7.23.6.*` will export all patch versions of the Mendix 8.12 runtime and the published build of the Mendix 7.23.6 runtime.
+4. Enter the version numbers of any Mendix Runtimes which you want to export. You can use wildcards and create a list of versions separated by commas. For example `8.12.*, 7.23.6.*` will export all patch versions of the Mendix 8.12 runtime and the published build of the Mendix 7.23.6 runtime. <!--Can we change the screenshot to newer versions? I can afterwards change the example in the text to match the screenshot-->
```
Comment (MariaShaposhnikova) [inline content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md]: @katarzyna-koltun-mx , I can update the text after you update the screenshot and use the new versions in the example.

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/deleting-objects.md [hunk 6]
```diff
@@ -109,10 +109,6 @@ In detail, the following graph shows what happens during deletions:
 
 {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/deleting-objects/during-deletes.png" class="no-border" >}}
 
-{{% alert color="warning" %}}
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/deleting-objects.md:112]: @Yiyun333 , is it OK to delete this or shall I keep it?

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/rollback-object.md [hunk 7]
```diff
@@ -97,6 +97,3 @@ Pressing a **Cancel** button or triggering a rollback object activity will initi
 
 {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/rollback-object/during-rollback.png" class="no-border" >}}
 
-{{% alert color="warning" %}}
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/rollback-object.md:100]: @Yiyun333 , is it OK to delete this or shall I keep it?

File: content/en/docs/refguide9/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/deleting-objects.md [hunk 8]
```diff
@@ -105,10 +105,6 @@ In detail, the following graph shows what happens during deletions:
 
 {{< figure src="/attachments/refguide9/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/deleting-objects/during-deletes.png" class="no-border" >}}
 
-{{% alert color="info" %}}
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide9/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/deleting-objects.md:108]: @Yiyun333 , is it OK to delete this or shall I keep it?

File: content/en/docs/refguide9/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/rollback-object.md [hunk 9]
```diff
@@ -96,7 +96,3 @@ Pressing a **Cancel** button or triggering a rollback object activity will initi
     * An object with the state **Instantiated** will be removed, and an object with any other state will be reverted back to the values it had during the last commit
 
 {{< figure src="/attachments/refguide9/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/rollback-object/during-rollback.png" class="no-border" >}}
-
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide9/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/rollback-object.md:99]: @Yiyun333 , is it OK to delete this or shall I keep it?

File: content/en/docs/support/security-findings-faq.md [hunk 10]
```diff
@@ -188,6 +188,7 @@ By default, end-users are allowed to have multiple sessions simultaneously. This
 
 ### 8.4 Bootstrap Vulnerabilities
```
Comment (MariaShaposhnikova) [inline content/en/docs/support/security-findings-faq.md]: @MarkvanMents , can I remove this section?

File: content/en/docs/deployment/mobileapp.md (hunk 4 above)
Comment (dbreseman) [reply content/en/docs/deployment/mobileapp.md]: @ConnorLand This is yours, right?

File: content/en/docs/control-center/security/set-up-sso-byoidp.md (hunk 3 above)
Comment (MarkvanMents) [reply content/en/docs/control-center/security/set-up-sso-byoidp.md]: Can be removed. @Karuna-Mendix FYI.

Comment (MarkvanMents) [reply content/en/docs/control-center/security/set-up-sso-byoidp.md]:
```suggestion
```

File: content/en/docs/support/security-findings-faq.md (hunk 10 above)
Comment (MarkvanMents) [reply content/en/docs/support/security-findings-faq.md]: Can remove this section, but should keep what is in 8.4.1.
Is it OK if I edit this section and commit to this branch?

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/deleting-objects.md (hunk 6 above)
Comment (Yiyun333) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/deleting-objects.md:112]: @MariaShaposhnikova I think we can remove the version info and keep the rest: 

> The **Committing** state of the **IMendixObject** is deprecated.

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/rollback-object.md (hunk 7 above)
Comment (Yiyun333) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/rollback-object.md:100]: See above.

File: content/en/docs/refguide9/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/deleting-objects.md (hunk 8 above)
Comment (Yiyun333) [reply content/en/docs/refguide9/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/deleting-objects.md:108]: See above.

File: content/en/docs/refguide9/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/rollback-object.md (hunk 9 above)
Comment (Yiyun333) [reply content/en/docs/refguide9/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/rollback-object.md:99]: See above.

File: content/en/docs/support/security-findings-faq.md (hunk 10 above)
Comment (MariaShaposhnikova) [reply content/en/docs/support/security-findings-faq.md]: @MarkvanMents , yes, please go ahead.

Comment (MarkvanMents) [reply content/en/docs/support/security-findings-faq.md]: Rewritten in https://github.com/mendix/docs/pull/8194/commits/b743986e1b682d624715f0047ef8fd849f1d7bf1

File: content/en/docs/deployment/mobileapp.md (hunk 4 above)
Comment (ConnorLand) [reply content/en/docs/deployment/mobileapp.md]: Checking now. We'll either need to change it to "Mendix 8.0 and above" or "Mendix 8.18 and above" (which is the Mx8 LTS).

File: content/en/docs/appstore/use-content/modules/forgot-password.md [hunk 11]
```diff
@@ -15,15 +15,14 @@ This module allows the end-user to enter their email address, and an email will
 Versions of the Forgot Password module below 4.1.0 (for Mendix 8) and 5.1.0 (for Mendix 9 and above) have a dependency on the deprecated [Email Module with Templates](https://marketplace.mendix.com/link/component/259/) module. Mendix recommends upgrading to the later version using the instructions in [Migrate from Email Module with Templates to Email Connector](#migrate-email) section below.
 {{% /alert %}}
```
Comment (Karuna-Mendix) [inline content/en/docs/appstore/use-content/modules/forgot-password.md:17]: Looks good to me.

File: content/en/docs/deployment/mobileapp.md (hunk 4 above)
Comment (ConnorLand) [reply content/en/docs/deployment/mobileapp.md]: I've fixed it and removed the Mx7 mention. https://github.com/mendix/docs/pull/8194/commits/5e7927f1e658b4b0dbfbe5849bb4a018e6deb9c2

File: content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md (hunk 5 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md]: @MariaShaposhnikova I've reached out to the team. If the issue is urgent, I suppose we can just remove the screenshot and list the versions in text. Let me know if that would be helpful.

File: content/en/docs/appstore/use-content/partner-solutions/apd/rg-apd/rg-three-apd/rg-three-recorder/rg-three-browser-recorder-results.md (hunk 2 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/use-content/partner-solutions/apd/rg-apd/rg-three-apd/rg-three-recorder/rg-three-browser-recorder-results.md:58]: Hi @MariaShaposhnikova , I checked with Danny. It is fine to delete this.

File: content/en/docs/appstore/use-content/partner-solutions/apd/rg-apd/rg-two-apm/rg-two-recorder/rg-two-browser-recorder-results.md (hunk 1 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/use-content/partner-solutions/apd/rg-apd/rg-two-apm/rg-two-recorder/rg-two-browser-recorder-results.md:58]: I checked with Danny. It is fine to remove the warning.
It seems that the screenshot does not cover the whole tab.

File: content/en/docs/appstore/use-content/modules/forgot-password.md (hunk 11 above)
Comment (Karuna-Mendix) [reply content/en/docs/appstore/use-content/modules/forgot-password.md:17]: Hi @MariaShaposhnikova, I found one more observation in this doc and I am waiting for confirmation from the team.

Comment (Karuna-Mendix) [reply content/en/docs/appstore/use-content/modules/forgot-password.md:17]: Hi @MariaShaposhnikova, I confirmed with the team and made changes into this [commit](https://github.com/mendix/docs/pull/8194/commits/e6ce6663493852465c6646f8def677a7b31fa450). Please review.

File: content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md (hunk 5 above)
Comment (katarzyna-koltun-mx) [reply content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md]: @MariaShaposhnikova After consideration, I've removed the screenshot for future compatibility. It is not crucial to understand the text and there's another similar screenshot in the same article.

Comment (MariaShaposhnikova) [reply content/en/docs/deployment/private-cloud/private-cloud-cluster/private-cloud-migrating.md]: @katarzyna-koltun-mx , thanks for the update! I will change the text in the example.

**Pull request merged**